### PR TITLE
Introducing statesman additions - it's different but nice

### PR DESCRIPTION
**Pull request by PragTob**

### CODE REVIEW

File: app/models/statesman1.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  def state
+    current_state
+  end
+end
+
+
+
+
+# class OrderStateMachine
```
Comment (bitboxer) [inline app/models/statesman1.rb]: What is this? Looks strange to have such a long comment blog with code :wink:

**Comment by bitboxer** [comment]
Only one little hint, the rest looks good.

File: app/models/statesman1.rb (hunk 1 above)
Comment (PragTob) [inline app/models/statesman1.rb]: Oh we had that in some of the other implementations as well, was the first usage exampled just copied over as a short API reference :)

We could remove them everywhere

Comment (bitboxer) [inline app/models/statesman1.rb]: Yep. I think a little cleanup would be nice.

**Pull request merged**